### PR TITLE
Assignment 4

### PR DESCRIPTION
**Pull request by suyash929**
@sriram15690 , please review Assignment 4 of Javascript

### CODE REVIEW

File: Assignment3/calculatingWithFunctions.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    return arguments[0](0)
+  }else{ // we will return integer
+    return 0
+  }
+
+}
+function one() {
+
+  if(arguments.length>0){
+    return arguments[0](1)
```
Comment (sriram15690) [inline Assignment3/calculatingWithFunctions.js:13]: You can user ternary as well. Use it next time.

return (arguments.length>0 ? arguments[0](0) :  0)

File: Assignment3/calculatingWithFunctions.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+  if(arguments.length>0){
+    return arguments[0](6)
+  }else{
+    return 6
+  }
+
+}
+function seven() {
+
```
Comment (sriram15690) [inline Assignment3/calculatingWithFunctions.js:66]: Functions one to Seven can be combined as one function. pls re-work

File: Assignment3/getMiddleCharacter.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+function getMiddle(stringValue)
+{
```
Comment (sriram15690) [inline Assignment3/getMiddleCharacter.js:2]: Can u google if there are better "getMiddle" methods?

**Review by sriram15690 — CHANGES_REQUESTED** [review]

File: Assignment3/calculatingWithFunctions.js (hunk 1 above)
Comment (suyash929) [reply Assignment3/calculatingWithFunctions.js:13]: Sure

File: Assignment3/calculatingWithFunctions.js (hunk 2 above)
Comment (suyash929) [reply Assignment3/calculatingWithFunctions.js:66]: We go the functions from KATA compiler , there functions were divided, we just had to add our code inside function. That's why we did like this

File: Assignment3/getMiddleCharacter.js (hunk 3 above)
Comment (suyash929) [reply Assignment3/getMiddleCharacter.js:2]: ok I will do it

**Pull request merged**